__tests__/clockSkew.js

```diff
@@ -101,7 +101,7 @@ describe('Clock skew', () => {
 
     await driver.wait(minNumActivitiesShown(3), timeouts.directLine);
 
-    const lastActivity = await driver.findElement(By.css('[role="list"] > li:last-child p'));
+    const lastActivity = await driver.findElement(By.css('.webchat__basic-transcript__activity:last-child p'));
```

Comment:
can you use `getTracnscript` here instead?

Reply:
Quality of midnight coding... fixed!